config/params/mainnet_config.go

```diff
@@ -254,6 +254,7 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	// Mevboost circuit breaker
 	MaxBuilderConsecutiveMissedSlots: 3,
 	MaxBuilderEpochMissedSlots:       5,
+	LocalBlockValueBoost:             10,
```

Comment:
This is the wrong place to set the default. Let's set it at the flag level so that it's clearly a default value of 10. 

Add `Value: 10` to the LocalBlockValueBoost flag here: https://github.com/prysmaticlabs/prysm/blob/f7f1d249f223151b4389f21fbd455f7047bc86b6/cmd/beacon-chain/flags/base.go#L28-L32